src/python/pants/bin/BUILD

```diff
@@ -103,6 +103,7 @@ target(
         "src/python/pants/backend/shell/lint/shfmt",
         "src/python/pants/backend/tools/preamble",
         "src/python/pants/backend/tools/taplo",
+        "src/python/pants/backend/tools/trufflehog",
```

Comment:
When you move the registration, change this appropriately.

Reply:
Done!